app/inputs/city_input.rb

```diff
@@ -0,0 +1,5 @@
+class CityInput < SimpleForm::Inputs::StringInput
```

Comment:
:heart:

I don't work with simple form much and thus I have never done custom inputs. While working on the character limited input the other day, I went that route, too. Never seen custom simple form inputs in years, now twice in two days :wink: